runelite-client/src/main/java/net/runelite/client/plugins/twitch/TwitchPlugin.java

```diff
@@ -26,14 +26,18 @@
 
 import com.google.common.base.Strings;
 import com.google.inject.Provides;
+
+import java.awt.*;
```

Comment:
oh no

Reply:
There is a way to set it so that you need something like 99 imports before inteliJ will shorten it to wildcards. I highly recommend it.

Reply:
Thank you I actually did not know about that, but I will definitely fix it.